packages/tuono/src/dynamic/route_lazy_loading.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import type { ReactElement } from 'react'
+
+import type { RouteComponent } from 'tuono-router'
+
+type ImportFn = () => Promise<{ default: RouteComponent }>
+
+export const __tuono__internal__lazyLoadRoute = (
```

Comment:
Additional note:
If the file exports a component named `RouteLazyLoading` the file should be have the same name.
For me using underscore_case for file housing React components is strange.

On `apps/documentation/src/components` they use kebab-case., 
but maybe there is a reason about the different naming casing between component file and function I don't know.

My opinion is that the file housing React components should have the same name of the component function, so they should use camelCase. 

WDYT?

Reply:
Agree. We should keep consistency and `PascalCase` is the best IMO

Reply:
#348